httpi.gemspec

```diff
@@ -19,11 +19,11 @@ Gem::Specification.new do |s|
   s.add_dependency 'rack'
   s.add_dependency 'socksify'
 
-  s.add_development_dependency 'rubyntlm', '~> 0.3.2'
-  s.add_development_dependency 'rake',     '~> 10.0'
-  s.add_development_dependency 'rspec',    '~> 2.14'
-  s.add_development_dependency 'mocha',    '~> 0.13'
-  s.add_development_dependency 'puma',     '~> 2.3.2'
+  s.add_development_dependency 'rubyntlm',    '~> 0.3.2'
+  s.add_development_dependency 'rake',        '~> 10.0'
+  s.add_development_dependency 'rspec',       '~> 3.4.0'
```

Comment:
Is there any reason specific to upgrade rspec here to 3.4.0?
I'm in favor to do this kind of change on another PR if you don't need anything specific from rspec 3.4.